plugins/main/bridge/bridge.go

```diff
@@ -96,19 +103,23 @@ func ensureBridgeAddr(br *netlink.Bridge, ipn *net.IPNet, forceAddress bool) err
 	return nil
 }
 
-func deleteBridgeAddr(br *netlink.Bridge, ipn *net.IPNet) error {
+func deleteBridgeAddr(br *netlink.Bridge, ipn *net.IPNet, isIPV4 bool) error {
 	addr := &netlink.Addr{IPNet: ipn, Label: ""}
 
-	if err := netlink.LinkSetDown(br); err != nil {
-		return fmt.Errorf("could not set down bridge %q: %v", br.Name, err)
+	if isIPV4 {
+		if err := netlink.LinkSetDown(br); err != nil {
+			return fmt.Errorf("could not set down bridge %q: %v", br.Name, err)
+		}
```

Comment:
Didn't see this before.  I know you're not adding it, but I really wonder why we bother with up/down.  I'm pretty sure you can remove an address without having to down the bridge...  @steveeJ any idea?